src/main/java/org/datavyu/plugins/nativeosx/NativeOSXMediaPlayer.java

```diff
@@ -21,21 +21,21 @@
   // The time seeked to will be within the bound [time-INFINITE, time+INFINITE]
   protected static final int NORMAL_SEEK_FLAG = 2;
 
-  protected static final int INITIAL_VOLUME = 100;
+  protected static final int INITIAL_VOLUME = 10;
```

Comment:
10 or 100?  Note, ffmpeg uses 100.

Reply:
Actually, AVFoundation range is from 0.0 to 1.0, I will have to change that and scale according to the ffmpeg volume range

Reply:
Please fix this.

Reply:
Done!